packages/custom-properties/test/__snapshots__/test.js.snap

```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`transforms a theme config to css variables 1`] = `
+Object {
+  "--color-accent": "#609",
+  "--color-background": "#fff",
+  "--color-dark-background-surface": "#fff",
+  "--color-dark-foreground-text": "#000",
+  "--color-muted": "#f6f6f6",
+  "--color-primary": "#07c",
+  "--color-secondary": "#05a",
+  "--color-text": "#000",
+  "--font-body": "system-ui, sans-serif",
+  "--font-heading": "system-ui, sans-serif",
+  "--font-monospace": "Menlo, monospace",
+  "--font-weight-body": 400,
+  "--font-weight-bold": 700,
+  "--font-weight-heading": 700,
+  "--line-height-0": 1.5,
+  "--line-height-1": 1.125,
+  "--size-0": "10em",
+  "--size-1": "20em",
+  "--size-2": "30em",
+  "--size-3": "40em",
+  "--space-0": 0,
+  "--space-1": 2,
+  "--space-2": 3,
+  "--space-3": 4,
+  "--space-4": 5,
+  "--space-5": 6,
+}
+`;
+
+exports[`transforms a theme config to css variables with prefix 1`] = `
+Object {
+  "--🍭-color-accent": "#609",
+  "--🍭-color-background": "#fff",
+  "--🍭-color-dark-background-surface": "#fff",
+  "--🍭-color-dark-foreground-text": "#000",
+  "--🍭-color-muted": "#f6f6f6",
+  "--🍭-color-primary": "#07c",
+  "--🍭-color-secondary": "#05a",
+  "--🍭-color-text": "#000",
+  "--🍭-font-body": "system-ui, sans-serif",
+  "--🍭-font-heading": "system-ui, sans-serif",
+  "--🍭-font-monospace": "Menlo, monospace",
+  "--🍭-font-weight-body": 400,
```

Comment:
Not sure what the best thing to do here is, and it might be an edge case. With hyphenated theme object keys, `fontWeights`, `fontSizes`, *could* collide with `fonts.size.body` (I doubt people would use this sort of naming convention)

Reply:
Is the confusion that they might be using `fonts.size.body` then `font-size-body`. I would prefer to keep it consistent despite the fact that it might not be logically correct. It also means one less dependency.

Reply:
Yeah, I'm suggesting the CSS name could be for e.g. `--fontSize-0` and `--fontWeight-bold` instead of hyphenating the original key. That way `-` is only used in place of `.`

Reply:
Sounds good to me. It also feels more consistent.

Reply:
I have made the change. It now keeps the camelCase key but still removes the plural.